src/index.js

```diff
@@ -1,5 +1,9 @@
-import React from "react";
 import ReactDOM from "react-dom/client";
+import "./style/reset.scss";
+import "./style/common.scss";
+import "./style/variables.scss";
+import "./style/mixin.scss";
+import Router from "./Router";
```

Comment:
```suggestion
import ReactDOM from "react-dom/client";
import Router from "./Router";
import "./style/reset.scss";
import "./style/common.scss";
```

import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)